app/controllers/concerns/reauthentication_required_concern.rb

```diff
@@ -38,31 +26,13 @@ def recently_authenticated?
     authn_at > Time.zone.now - IdentityConfig.store.reauthn_window
   end
 
-  def prompt_for_current_password
-    store_location(request.url)
-    user_session[:context] = 'reauthentication'
-    user_session[:factor_to_change] = factor_from_controller_name
```

Comment:
This is the only place this is assigned, but it's still referenced in a view? 🤔 

https://github.com/18F/identity-idp/blob/77311e93caf0d0ee081267471b14233ff08abcaa/app/views/mfa_confirmation/new.html.erb#L7

Reply:
Yeah, this controller has been re-used in other ways where it's not changing factors and the change_factor does not exist. It's also not translated and the content should be re-written to be more generic.

Reply:
https://gsa-tts.slack.com/archives/CEUQ9FXNJ/p1643994476402979 is a Slack thread that discusses it a bit (with an associated ticket that's linked above that line)

Reply:
I'll un-icebox that ticket and get it in front of the team.